luskan/src/main/java/org/opencadc/luskan/AdqlQueryImpl.java

```diff
@@ -115,6 +116,12 @@ protected void init() {
         TableNameReferenceConverter tnrc = new TableNameReferenceConverter(tnc.map);
         super.navigatorList.add(new SelectNavigator(new ExpressionNavigator(), tnrc, tnc));
 
+        // add IS NOT NULL constraint for artifact.storagelocation_storageid when querying storage sites
+        MultiValuedProperties properties = getProperties();
+        String isStorageSite = properties.getFirstPropertyValue(LuskanConfig.STORAGE_SITE_KEY);
+        if (isStorageSite != null) {
```

Comment:
Since the README says
```
org.opencadc.luskan.isStorageSite={true|false}
```
Shouldn't this be something like
```
boolean isStorageSite = Boolean.parseBoolean(...);
if (isStorageSite) {
...
```
As it is, `false` and 'flibble` will also make `if (isStorageSite != null) {` true.

Reply:
Wasn't sure if this property should be mandatory at first so did the null check. Forgot to revisit after decided it should be, but parseBoolean is the proper way to get the property value.